src/ruby/yast/i18n.rb

```diff
@@ -1,6 +1,11 @@
 require "fast_gettext"
 require "logger"
 
+# just to make the y2makepot script happy
+'
+textdomain "example"
+'
+
```

Comment:
well, ok, but please place it to bottom of file and not top, as it is just workaround.